### PR TITLE
#48: Input file path is now correctly populated.

### PR DESCRIPTION
**Pull request by schrodingersket**

### CODE REVIEW

File: lib/Inkscape.js [hunk 1]
```diff
@@ -93,26 +93,31 @@ class Inkscape extends Stream {
...
-        setOutputFormat = false;
-      } else if (setInputFormat) {
-        this.inkscapeInputFilePath = getTemporaryFilePath({
-          suffix: `.${this.inputFormat || 'svg'}`,
-        });
-        this.inkscapeArgs.push(this.inkscapeInputFilePath);
-        setInputFormat = false;
+    // Set output format as applicable
+    //
+    if (setOutputFormat || !!outputFormat) {
```
Comment (papandreou) [inline lib/Inkscape.js:100]: Unnecessary `!!` as `if` conditions are already evaluated in a boolean context:

```suggestion
    if (setOutputFormat || outputFormat) {
```

File: lib/Inkscape.js [hunk 2]
```diff
@@ -93,26 +93,31 @@ class Inkscape extends Stream {
...
-    });
+      this.inkscapeOutputFilePath = getTemporaryFilePath({
+        suffix: `.${this.outputFormat}`,
+      });
+      setOutputFormat = false;
+    }
 
+    // Set input format as applicable
+    //
+    if (setInputFormat || !!inputFormat) {
```
Comment (papandreou) [inline lib/Inkscape.js:114]:
```suggestion
    if (setInputFormat || inputFormat) {
```

File: yarn.lock [hunk 3]
```diff
@@ -0,0 +1,2780 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (papandreou) [inline yarn.lock:1]: Please remove. This project does not use yarn.

**Review by papandreou — CHANGES_REQUESTED** [review]
Looks good, just some small nits.

File: lib/Inkscape.js (hunk 1 above)
Comment (schrodingersket) [reply lib/Inkscape.js:100]: For sure. Vim'd too fast on the copy/paste of these guys; I usually will add `!!` on variable assignment to keep the typing consistent, which is why that was there in the first place. Thanks for catching that. (:

File: yarn.lock (hunk 3 above)
Comment (schrodingersket) [reply yarn.lock:1]: My bad. Used to adding a lockfile on projects since I mostly work with applications instead of libraries.

**Comment by schrodingersket** [comment]
Something got borked on the linting here, but my system isn't showing enough output for me to figure out what it was. Any chance you might have insight there? Here's what I get:

```
➜  node-inkscape git:(bugfix/48) ✗ npm run lint

> inkscape@3.1.0 lint /work/jonathan/n9/node-inkscape
> eslint . && prettier --check '**/*.{js,md}'

Checking formatting...
[warn] test/Inkscape.js
[warn] Code style issues found in the above file(s). Forgot to run Prettier?
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! inkscape@3.1.0 lint: `eslint . && prettier --check '**/*.{js,md}'`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the inkscape@3.1.0 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

I don't see anything jumping out at me in `test/Inkscape.js` :/

**Comment by schrodingersket** [comment]
Something got borked on the linting here, but my system isn't showing enough output for me to figure out what it was. Any chance you might have insight there? Here's what I get:

```
➜  node-inkscape git:(bugfix/48) ✗ npm run lint

> inkscape@3.1.0 lint ~/node-inkscape
> eslint . && prettier --check '**/*.{js,md}'

Checking formatting...
[warn] test/Inkscape.js
[warn] Code style issues found in the above file(s). Forgot to run Prettier?
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! inkscape@3.1.0 lint: `eslint . && prettier --check '**/*.{js,md}'`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the inkscape@3.1.0 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

I don't see anything jumping out at me in `test/Inkscape.js` :/

**Comment by papandreou** [comment]
It needs `prettier --write test/Inkscape.js`

If you're using vscode, I can highly recommend this extension: https://github.com/prettier/prettier-vscode

My settings look like this:

```
    "editor.formatOnSave": true,
    "editor.defaultFormatter": "esbenp.prettier-vscode",
    "prettier.requireConfig": true,
```

The last line means that the extension doesn't do anything unless the project has a prettier configuration file, so it won't be in the way if you're mainly working on projects that aren't using prettier.

**Comment by schrodingersket** [comment]
Got it. I went ahead and added a `prettier` NPM run command, if that's alright; I'm just using Vim/command line, and that allows for a nice little pre-commit Git hook (:

**Review by papandreou — APPROVED** [review]
LGTM :rocket:

**Comment by papandreou** [comment]
Released in 3.1.1

**Pull request merged**